### ISSUE TITLE
Drawing squares to the canvas is slow

### ISSUE THREAD

**Issue by K0-RR** [Jelmerro/crossdirstat#7]
5 minutes or more to scan root compared to ~30 seconds of Jdiskreport and ~15 seconds of Filelight.

Generating canvas was responsible for about 90% of the process.

**Comment by Jelmerro** [comment]
I agree that it's not fast, but I also don't see how I can speedup the canvas drawing. If you do think it can be improved, feel free to do so and make a PR for it :slightly_smiling_face:.  The function that actually draws the squares can be found [right here](https://github.com/Jelmerro/crossdirstat/blob/master/app/js/visual.js#L222).

**Comment by Jelmerro** [comment]
I managed to improve this a lot by moving to offscreencanvas, so I consider this fixed in 2.5.0 :tada:
